public/js/actions/types.js

```diff
@@ -88,7 +87,7 @@ type SourceAction =
                sourceText: SourceText }}
   | { type: "CLOSE_TAB", id: string };
 
-type UIAction = { type: constants.TOGGLE_FILE_SEARCH, searchOn: boolean };
+type UIAction = { type: "TOGGLE_FILE_SEARCH", searchOn: boolean };
```

Comment:
Yeah, good idea. This file could likely be updated to use the constants in its own PR.